CAP/gap/UniversalObjects.gd

```diff
@@ -1422,6 +1422,66 @@ DeclareOperationWithCache( "MorphismBetweenDirectSums",
 DeclareOperationWithCache( "MorphismBetweenDirectSumsOp",
                            [ IsList, IsInt, IsInt, IsCapCategoryMorphism ] );
 
+#! @Description
```

Comment:
Do those primitives violate the fact, that the diagram should be in the first place, i.e., should the object list be the first argument?

Reply:
I think we only have this convention for those basic operations concerning the data defining a limit (or colimit):
the limit object, the diagram data, the universal property data.
In `ComponentOfMorphismIntoDirectSum` and `ComponentOfMorphismFromDirectSum`  I like to think of the first argument as the most relevant one.

Reply:
Ok.